TombEngine/Scripting/Internal/TEN/Misc/Miscellanous.h

```diff
@@ -1,4 +1,16 @@
 #pragma once
+#include "Game/camera.h"
+
+static const std::unordered_map<std::string, CameraType> kCameraType
```

Comment:
You need to move enum to a separate file and also add ldoc comments to it, so it appears in documentation. For reference, look into `ActionIDs.h` file.